index.html

```diff
@@ -244,6 +244,20 @@ <h1>I'm Naan.
                     </div>
                 </div>
             </div>
+            <section class="contact_me">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+
+                <form action="https://formspree.io/f/xvonabyg"
+                method="POST">
+                    <ul>
+                        <li><input value="" placeholder="Enter your name" id="name" type="text" name="user_name" maxlength="30" required></li>
+                        <li><input value="" placeholder="Enter your email" id="email" type="email" name="email" required></li>
+                    </ul>
+                    <textarea value="" id="message" placeholder="Write your message here" type="text" name="message" maxlength="500" required></textarea>
+                    <button type="submit">Get in touch</button>
```

Comment:
- [ ] [OPTIONAL] I think you can choose to remove the `type` attribute for this case because the default value for the `type` attribute is `submit`. 😀